src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](http://etheruem.org) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
+
+## Who can submit content to Ethereum.org {#who-can-help}
+
+Anyone! [Ethereum.org](http://ethereum.org) is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
+
+## Audience {#audience}
+
+[Ethereum.org](http://ethereum.org) content caters to a large and varied audience. Loosely we can categorize them as:
+
+#### Individuals
+
+App users, investors, enthusiasts, people who are "new to Ethereum"
+
+#### Developers
+
+Developers or others who want technical information about Ethereum.
+
+#### Enterprises
+
+People, businesses, and other organizations who want to understand Ethereum's value in an enterprise setting
+
+Because [Ethereum.org](http://ethereum.org) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
+
+## Objectivity {#objectivity}
+
+[Ethereum.org](http://ethereum.org) documentation aims to maintain a credibly neutral source of truth to inform readers about Ethereum and its ecosystem. Some examples of things that we don't want in the content on Ethereum.org:
+
+**Grand, unverifiable claims about Ethereum or adjacent technologies**
+
+> e.g. _ "Ethereum will take over the world because..."_
+
+**Hostile or confrontational language aimed at any organization or person**
+
+> e.g. _ "Company X is bad because they are centralized!"_
+
+**Politically charged rhetoric**
+
+> e.g. _ "This political party are better for decentralization because..."_
+
+## Tone {#tone}
+
+The tone of the content on the site should be welcoming, friendly and authoritative. Where possible, jargon should be minimized and simple language used instead.
+
+## Best Practices {#best-practices}
+
+### Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
+
+### Acroynyms {#acroynyms}
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
```

Comment:
This goes to a point I made on #3653... should we encourage that the abbreviation be used after the term has been introduced on any given page?

Reply:
Commented in #3653 but I lean slightly towards using the full term